knowledge/science/botany/dianthus_caryophyllus/qna.yaml

```diff
@@ -0,0 +1,20 @@
+created_by: 
+domain: flowers
+seed_examples:
+- answer: canation or clove pink
+  question: what is the common name for Dianthus Caryophyllus?
+- answer: Along with the Red Rose the red carnation can be used as a symbol of socialism and the labour movement, and historically has often been used in demonstrations on International Workers Day (May Day).
+  question: What can Dianthus Caryophyllus be used as a symbol for?
+- answer: they were mentioned in greek literature 2,000 years ago
+  question: how old are dianthus caryophyllus?
+- answer: Dianthus caryophyllus is a herbaceous perennial plant growing up to 80 cm (31
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!